system-addon/lib/TopSitesFeed.jsm

```diff
@@ -293,6 +335,9 @@ this.TopSitesFeed = class TopSitesFeed {
       case at.TOP_SITES_UNPIN:
         this.unpin(action);
         break;
+      case at.SCREENSHOT_REQUEST:
+        this.setCustomTopsiteScreenshot(action.data);
```

Comment:
This is called on a preview, so calling a `set` would seem to be more permanent than a preview. So either the method is not actually permanent or it shouldn't be as permanent?

Reply:
s/set/get since the function now calls `getScreenshotForURL`.